src/Sylius/Bundle/ShopBundle/Router/LocaleStrippingRouter.php

```diff
@@ -57,11 +57,13 @@ public function getRouteCollection(): RouteCollection
         return $this->router->getRouteCollection();
     }
 
-    public function warmUp($cacheDir): void
+    public function warmUp($cacheDir): array
```

Comment:
As it returns an array of strings, add `string[]` as a return type in the phpdoc, please.